integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala

```diff
@@ -84,15 +85,7 @@ object MojoImplementation {
         val suffix = if (classifier.nonEmpty) s":$classifier" else ""
         log.info("Resolving artifact: " + artifact + suffix)
         val request = new ArtifactRequest()
-        request.setArtifact(
-          new DefaultArtifact(
-            artifact.getGroupId(),
-            artifact.getArtifactId(),
-            classifier,
-            artifact.getType(),
-            artifact.getVersion()
-          )
-        )
+        request.setArtifact(RepositoryUtils.toArtifact(artifact))
```

Comment:
Reading https://github.com/apache/maven/blob/master/maven-core/src/main/java/org/apache/maven/RepositoryUtils.java#L56-L60 I can see it's an internal tool and shouldn't be used, hence I will revert it.

Will need to duplicate some of its logic.

Reply:
Removed in 8a606df